src/utils/Endpoint.js

```diff
@@ -0,0 +1,41 @@
+import transformToHarRequest from './transformToHarRequest';
+import { HTTPSnippet } from 'httpsnippet';
+
+const supportedClients = {
+  curl: {
+    targetId: 'shell',
+    options: { short: true, indent: ' ', prettifyJson: true },
+  },
+};
+
+class Endpoint {
+  constructor(props) {
+    this.path = props.path;
+    this.method = props.method;
+    this.requests = props.requests;
+  }
+
+  static create(data) {
+    const harRequest = transformToHarRequest(data);
+
+    const snippet = new HTTPSnippet(harRequest);
+    const path = snippet.requests[0].uriObj.pathname;
+    const method = snippet.requests[0].method;
```

Comment:
@MeganSteenkamp Suggestion: Use `path` and `method` from `data instead of relying on the HAR output